source/features/index.tsx

```diff
@@ -164,13 +164,9 @@ const setupPageLoad = async (id: FeatureID, config: InternalRunConfig): Promise<
 // with the current version of extension
 const checkForHotfixes = cache.function(async () => {
 	const response = await fetch('https://raw.githubusercontent.com/sindresorhus/refined-github/hotfix/hotfix.json');
-	const hotfixes = await response.json();
+	const hotfixes: AnyObject | false = await response.json();
```

Comment:
should this have the `| false`?

trying to resolve a `@typescript-eslint/prefer-optional-chain` error on L169 threw me : 
```
source/features/index.tsx:167:16 - error TS2339: Property 'unaffected' does not exist on type 'false | Record<string, any>'.
  Property 'unaffected' does not exist on type 'false'.

169  if (hotfixes?.unaffected) {
                   ~~~~~~~~~~

source/features/index.tsx:169:32 - error TS2339: Property 'unaffected' does not exist on type 'false | Record<string, any>'.
  Property 'unaffected' does not exist on type 'false'.

171  	if (compareVersions(hotfixes.unaffected, currentVersion) >= 0) {
     	                             ~~~~~~~~~~
```

I'm not an expert with TypeScript, but I believe we should always expect `hotfixes` to be a JSON object right? Even if we do not have any breaking features, we'll probably have `{ false }` in the file. I don't know what would happen if there was a network error though.

I was able to avoid the lint error locally by removing the `| false`. Hence the question.

Reply:
If eslint complains about `hotfix && hotfix.unaffected`, that rule is wrong in this case; disable it with `eslint-disable-next-line`

Reply:
`{false}` is not a valid JSON object (or even valid js). `false` is. The file will literally only contain the word `false`, hence the type `AnyObject | false`